### PR TITLE
Cleaning up some details

### PR DESCRIPTION
**Pull request by saskiad**

### CODE REVIEW

File: schemas/Procedures Schema.json [hunk 1]
```diff
@@ -30,6 +30,20 @@
             "items": {"$ref": "#/$defs/prep"},
             "uniqueItems": true
         },
```
Comment (dyf) [inline schemas/Procedures Schema.json:32]: Some funky spacing here

**Review by dyf — CHANGES_REQUESTED** [review]
looks fine, modulo some inconsistent spacing. do any examples need updating?

File: schemas/Procedures Schema.json (hunk 1 above)
Comment (saskiad) [reply schemas/Procedures Schema.json:32]: examples need updating - I will work on that soon (might not be until tomorrow)

Comment (saskiad) [reply schemas/Procedures Schema.json:32]: Every editor I open this in shows it with non-funky spacing.... why is it funky here?

Comment (saskiad) [reply schemas/Procedures Schema.json:32]: found it

Comment (dyf) [reply schemas/Procedures Schema.json:32]: looks like tabs vs spaces

**Review by dyf — APPROVED** [review]

**Pull request merged**